### PR TITLE
convert cpuset.Builder from struct to pointer

### PR DESCRIPTION
**Pull request by xiaoanyunfei**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

 /kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #https://github.com/kubernetes/kubernetes/issues/81010

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```
/sig node

### CODE REVIEW

**Review by mattjmcnaughton — APPROVED** [review]
Overall, I think this change makes sense and appreciate the additional test coverage. Thanks!

File: pkg/kubelet/cm/cpuset/cpuset_test.go [hunk 1]
```diff
@@ -36,6 +36,10 @@ func TestCPUSetBuilder(t *testing.T) {
 	if len(elems) != result.Size() {
 		t.Fatalf("expected cpuset %s to have the same size as %v", result, elems)
 	}
+	b.Add(6)
```
[the remark this answers is not in the document]
Comment (xiaoanyunfei) [reply pkg/kubelet/cm/cpuset/cpuset_test.go]: good advice，thanks

File: pkg/kubelet/cm/cpuset/cpuset.go [hunk 2]
```diff
@@ -44,7 +44,7 @@ func NewBuilder() Builder {
 
 // Add adds the supplied elements to the result. Calling Add after calling
 // Result has no effect.
-func (b Builder) Add(elems ...int) {
+func (b *Builder) Add(elems ...int) {
 	if b.done {
```
[the remark this answers is not in the document]
Comment (xiaoanyunfei) [reply pkg/kubelet/cm/cpuset/cpuset.go]: I add a log in case caller doesn't check it

**Comment by xiaoanyunfei** [comment]
> I see that this PR failed pull-kubernetes-verify in the staticcheck test:
> https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/82016/pull-kubernetes-verify/1166523444238487554
> 
> The failure is reported on a file this PR did not touch. I am running into a similar issue in my PR #81871 where staticcheck reports failures on files I did not modify. Any idea what is going on here?
I don't know. It's successful when I runned it locally
/retest

**Pull request closed without merging**